searches/linear_search.py

```diff
@@ -29,7 +29,9 @@ def linear_search(sequence: list, target: int) -> int:
     """
     for index, item in enumerate(sequence):
         if item == target:
-            return index
+            return (
+                index + 1
+            )  # The final output should not be based on indexing as it starts from 0
```

Comment:
```suggestion
            return index
```
Why change it to `index + 1`? That breaks the function because all outputs would be off by 1. I'm not sure what you mean when you say that the final output _shouldn't_ be based on the index.